### PR TITLE
Updated 'What to expect' and other content changes

### PR DESCRIPTION
**Pull request by allisonnorman**
- Added 'what to expect' content from Amanda 
- Updated home page ledes
- Added subtitle under "what to expect" on homepage (I asked Nick to provide more feedback on this as part of his work on that section)

Still to do, which might be better for @hursey013 to do: 
- Link to the 'what to expect' content section in top nav
- Remove subtopic from 'what to expect' section (Note: @hursey013 - Can we make it so that we don't have subtopics for this section? Ideally there are no jump downs within this section, but we could always get Amanda to write something else. For now, I have "what to expect" again as a placeholder.)

### CODE REVIEW

**Comment by allisonnorman** [comment]
Updates for #109

**Comment by allisonnorman** [comment]
**Note:** I was messing around with cards.js to test something earlier and though I undid it and couldn't see it locally, it was pushed in the last commit. I went ahead and changed it again, then undid, and pushed again and it seems to be removed now. Might be worth double checking that.

File: src/components/Home.js [hunk 1]
```diff
@@ -67,6 +67,10 @@ class Home extends Component {
               <h3 css={tw`text-4xl font-light text-gray-700`}>
                 What to expect
               </h3>
+              <h4 css={tw`text-xl font-light text-gray-700`}>
```
Comment (hursey013) [inline src/components/Home.js]: Semantically this isn't really a heading, so we can keep it visually the same while changing the markup:
```suggestion
              <span css={tw`block text-xl font-light text-gray-700`}>
```

File: src/components/Home.js [hunk 2]
```diff
@@ -67,6 +67,10 @@ class Home extends Component {
...
+                Treatment works, help is available, and people recover every
+                day.
+              </h4>
```
Comment (hursey013) [inline src/components/Home.js]:
```suggestion
              </span>
```

File: src/components/Home.js (hunk 1 above)
Comment (allisonnorman) [reply src/components/Home.js]: Change made

File: src/components/Home.js (hunk 2 above)
Comment (allisonnorman) [reply src/components/Home.js]: Change made

**Comment by allisonnorman** [comment]
@hursey013 - Made those changes. Do you think you could do the other two things and we can push this all at once or would you prefer for me to just keep the issue open for those?

- Link to the Getting started content section in top nav
- Remove subtopic from Getting Started section (Note: @hursey013 - Can we make it so that we don't have subtopics for this section? Ideally there are no jump downs within this section, but we could always get Amanda to write something else. For now, I have "what to expect" again as a placeholder.)

**Comment by allisonnorman** [comment]
Looks good to me! Feel free to merge, @hursey013

**Pull request merged**